scss/_spinners.scss

```
@@ -34,6 +34,7 @@
   }
   50% {
     opacity: 1;
+    transform: scale(1);
```

Probably better to change this to `transform: none;`, not gonna push this, since I can not reproduce and check the issue. @mdo, could you test that?
```suggestion
    transform: none;
```